src/brokers/broker-details/components/Configuration/Configuration.component.tsx

```diff
@@ -1,8 +1,8 @@
 import {
-  Split,
-  SplitItem,
+  // Split,
+  // SplitItem,
```

Comment:
best practice: never leave commented code. Code that we remove is always there in the history.